lib/uploadcare/rails/action_view/uploadcare_include_tags.rb

```diff
@@ -25,7 +25,7 @@ module UploadcareWidgetTags
         #     valid options: 'full', 'default', 'ie8', 'api', 'lang.en'
         #   min: (true/false, default: true) - sets which version to get, minified or not
 
-        CDN_HOSTNAME = ENV.fetch('UPLOADCARE_CDN_HOSTNAME', 'ucarecdn.com')
+        CDN_HOSTNAME = ENV.fetch('UPLOADCARE_CDN_HOSTNAME', 'ucarecdn.com').freeze
```

Comment:
Can we fetch this from environment in the config?
(so all configuration options will be in the config)